components/studio/charts/apps/rstudio/chart/templates/np.yaml

```
@@ -0,0 +1,24 @@
+
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ .Release.Name }}-any-external
+  namespace: {{ .Values.namespace }}
+spec:
+  podSelector:
+    matchLabels:
+      release: {{ $.Release.Name }}
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - {}
+  egress:
+    - to:
+        - ipBlock:
+            cidr: 0.0.0.0/0
+            except:
+              - 10.0.0.0/8
+              - 192.168.0.0/16
+              - 172.16.0.0/20 
```

```suggestion
              - 172.16.0.0/20
    - ports:
        - protocol: UDP
          port: 53              
```
Adding support for DNS lookups; might solve the "no internet" issue